CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,133 @@
+
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
```

Comment:
this is fantastic — I think it might benefit from stating clearly at the top which community we're referencing (which I assume is Alchemy overall, not just the contributors/users of this library). so we could just have this say "...make participation in the Alchemy community..." here, and the rest can continue to say e.g. "our community"

Reply:
Good callout. I might move this to a central location, like alchemyrpg/code-of-conduct and then link to it here and in our other repos. 🤔

Reply:
Yeah, I decided to centralize this so we could refer to it in other projects. Updated the CoC here accordingly to link out to that.